CONTRIBUTING.md

```diff
@@ -1,4 +1,49 @@
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
+
 ## Contributing to this Project
 
 OpenSearch is a community project that is built and maintained by people just like **you**.
 [This document](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md) explains how you can contribute to this and related projects.
+The brief summary below outlines some of the ways you can contribute.
+
+_Note: In any case where this document conflicts directly with the policies outlined by the main [CONTRIBUTING.md](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md) document for OpenSearch, the latter will always hold precedent._
+
+# Forms of Contribution
```

Comment:
Checkout .github CONTRIBUTING.md [1] it looks like there is a lot of overlap here

[1] https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#ways-to-contribute

Reply:
Yeah I pointed people towards the main document but was not sure if it was necessarily bad to have it written in two places. You may feel otherwise in which case I can move it but thought that it may be helpful for someone who just wanted to be squarely in security.

Reply:
When adding clarity - those changes should go onto the .github project.  When we are doing things as the security plugin we should only diverge when we have good reason.

Looking at the triage guidance [1] its manly about tags, compared to what you are proposing conducting triage meetings on Zoom with our agenda and process documented is repo specific.  I think there are some questions that would be really good to answer around the triage meeting that you could answer here that draw from your outreach document

- Do you need to go do triage for my issue to be triaged?
- What is the agenda of the triage meeting / how do I join the meeting?
- Why attend the triage meeting?


[1] https://github.com/opensearch-project/.github/blob/787d9715b507a4d55ac44502117c9017a9b2f456/MAINTAINERS.md?plain=1#L55-L61